pkg/sql/rename_table.go

```diff
@@ -286,25 +286,25 @@ func (p *planner) dependentFunctionError(
 	return sqlerrors.NewDependentBlocksOpError(op, typeName, objName, "function", fnDesc.GetName())
 }
 
-func (p *planner) dependentViewError(
+func (p *planner) dependentRelationError(
 	ctx context.Context,
 	typeName, objName string,
 	parentID descpb.ID,
-	viewDesc catalog.TableDescriptor,
+	desc catalog.TableDescriptor,
 	op string,
 ) error {
-	viewName := viewDesc.GetName()
-	if viewDesc.GetParentID() != parentID {
-		viewFQName, err := p.getQualifiedTableName(ctx, viewDesc)
+	viewName := desc.GetName()
+	if desc.GetParentID() != parentID {
+		viewFQName, err := p.getQualifiedTableName(ctx, desc)
 		if err != nil {
-			log.Warningf(ctx, "unable to retrieve name of view %d: %v", viewDesc.GetID(), err)
+			log.Warningf(ctx, "unable to retrieve name of relation %d: %v", desc.GetID(), err)
 			return sqlerrors.NewDependentObjectErrorf(
-				"cannot %s %s %q because a view depends on it",
-				op, typeName, objName)
+				"cannot %s %s %q because a %q depends on it",
```

Comment:
super nit: i think we should use `because a %s depends on it` (instead of `%q`). we don't need to quote the type of the object. also, we can cast `op` and `desc.GetObjectTypeString()` to `redact.SafeString` to avoid unnecessary redaction.